### PR TITLE
lint, header: Replace addlincese with go-header

### PR DESCRIPTION
**Pull request by qinqon**
The addlincese tool does not do a proper check of headers and does not
allow to enforce a custom time with block comments. This change replace
it with go-header that allow all of this, although we miss the headers
fix.

### CODE REVIEW

File: header.tpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ * This file is part of the NMPolicy project
```
Comment (EdDev) [inline header.tpl:1]: I do not think we need this line... Placing the one on line 15 here should do the job.

**Review by EdDev — COMMENTED** [review]
Only a small comment on the format.

File: header.tpl [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ * Copyright {{ YEAR }} NMPolicy Authors.
```
Comment (EdDev) [inline header.tpl:1]: I hope that next year, it will not fail when validating the older files.

**Review by EdDev — APPROVED** [review]
Thank you!

File: header.tpl (hunk 2 above)
Comment (qinqon) [reply header.tpl:1]: shit... https://github.com/denis-tingaikin/go-header/blob/9edc16b671fc30ca3a30727ae20a474fef305f7c/README.md#bult-in-values

Comment (qinqon) [reply header.tpl:1]: Let's just use a regex there.

```
regexp:
    key2: value2 # regexp value just checks regex match. The value should be a valid regexp pattern. Note `key2` should be used in template string as {{ key2 }} or {{ KEY2 }}.
```

Comment (EdDev) [reply header.tpl]: Based on the documentation, `{{year-range}}` would have done it.

Comment (qinqon) [reply header.tpl]: Nope, it uses time.Now for the right side of the range.

**Pull request merged**